README.md

```diff
@@ -83,9 +83,9 @@ The overall process is as follows.
    - Submit your package to the SailfishOS:Chum testing repository `sailfishos:chum:testing` by using the "Submit package" action of OBS. A version has to be specified when submitting.
    - The `sailfishos:chum:testing` maintainers will accept or reject your request and check if your software package successfully builds at the SailfishOS:Chum testing repository. If all is fine, the package will be promoted to the main `sailfishos:chum` repository by the SailfishOS:Chum maintainers. If something went wrong, you will be notified to resolve the issue.
 2. After a successful initial submission, you will be made maintainer of your software package in the `sailfishos:chum:testing` repository, which allows you to handle updates in a simplified manner:
-   - For updating your package, update the source, then change the version information in the OBS service file at `sailfishos:chum:testing`.
-   - Check that your software package successfully builds at the SailfishOS:Chum testing repository `sailfishos:chum:testing`.
-   - Ultimately use the "Submit package" action to trigger promoting your package from `sailfishos:chum:testing` to `sailfishos:chum`.
+   - For updating your package, prepare the source, then update the version information accordingly in the OBS service file `_service` at `sailfishos:chum:testing`: This will trigger a new build at OBS.
+   - Check that your software package successfully builds at the SailfishOS:Chum testing repository. Note that this needs some patience and might require reloading the browser window or using `osc blt` at the command line to observe that OBS progresses.
+   - Ultimately use the "Submit package" action to trigger promoting your package from `sailfishos:chum:testing` to `sailfishos:chum`: Fill the form with `sailfishos:chum` as the target repository, your regular package name as target package and the description, which is only for this submit request, with for example, `<package name> <version number>`.
```

Comment:
In practice, it is sufficient just to submit package by specifying "sailfishos:chum". No need for version or anything else unless you plan to change version immediately. Package name stays the same anyway, so no need to specify it either.

Reply:
Thank you @rinigus, this is exactly the guidance I needed; both, in practice ("What do I have to insert into these fields, are they mandatory or optional, and which scope does the description have?") and for enhancing this guide.

I will rephrase it using this information.

Reply:
I hope [commit ff11eae](https://github.com/sailfishos-chum/main/pull/68/commits/ff11eae9f238a328c88597c726be52f42c5b8ed1) does address this well.

Reply:
@rinigus, this review comment thread you started is still shown as "Changes requested" below (right above empty comment field at the very bottom), maybe this is why @piggz does not approve this PR.
When I hit the "Resolved" button for this comment thread, that does not change this, I assume somebody with maintainer's rights need to do that.

Hence I re-requested your review "formally".